content/docs/handling-events.md

```diff
@@ -58,6 +58,10 @@ Here, `e` is a synthetic event. React defines these synthetic events according t
 
 When using React you should generally not need to call `addEventListener` to add listeners to a DOM element after it is created. Instead, just provide a listener when the element is initially rendered.
 
+> Note:
+>
+> For performance reasons, React events are not guaranteed to be installed on the element they are created on. This means that using `stopPropagation` from event handlers installed outside of React might not be able to stop React events from occurring and vice versa. 
```

Comment:
I'm not sure about the wording "_installed on_" or "_installed outside of_".

Reply:
I was looking for a good word here. I am not happy with "installed" myself, but it was the best I could come up with. Suggestions welcome!

Reply:
(Sorry. Hit enter before I intended to.) What do you think about something like this?

> Note:
>
> React uses event delegation for performance reasons. Generally this is not observable but it is worth noting if you also use the browser's event system directly (via `addEventListener`). For example, calling `stopPropagation()` on a synthetic event will not prevent a native event handler from being executed.

Reply:
(I'm also not sure about the wording here. Just offering an idea.)

Reply:
Friendly ping 😄

Reply:
@bvaughn Sorry for the late reply. I like your version better than mine, although I needed a bit to understand what "observable" meant in this context. I was looking for a technical meaning. But that is bikeshedding territory.

Reply:
No problem. Want to take a stab at updating the wording then?